Guide/scripts.markdown

```diff
@@ -64,6 +64,56 @@ This is made possible because of the [she-bang line](https://en.wikipedia.org/wi
 
 In case you get a permission error, try to add the executable flag via `chmod +x Application/Script/HelloWorldToAllUsers.hs`.
 
+## Running a script from ghci
+
+You can also open a ghci prompt to test features in scripts interactively.
+
+```bash
+make ghci
+```
+
+Then you can load your script into the interpreter:
+```
+:l Application/Scripts/MyScript
+```
+
+and run the script from the IHP ghci command line:
+
+```haskell
+IHP> runScript ihpDefaultConfig runTestName
+```
+
+The `ihpDefaultConfig` is made available from the `Application.Script.Prelude` import but can be substituted
+with your own `Config` data structure. This is particularly useful for adjusting logging levels or
+testing new APIs that require certain configuration variables.
+
+You can define a custom configuration (or import one from Config.hs) in your Script
```

Comment:
I think this is too generic. I rather want this documentation to show specifically on how to import Config.hs and then use that configuration for running the script as that's the most common case (and good docs need to be specific).

Reply:
Updated to show the import in a script and then running it from ghci.